src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,47 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Drivetrain extends SubsystemBase {
+  CANSparkMax Leftmaster = new CANSparkMax(Constants.DRIVETRAIN_LEFTMASTER_ID, MotorType.kBrushless);
+  CANSparkMax Left1 = new CANSparkMax(Constants.DRIVETRAIN_LEFT1_ID, MotorType.kBrushless);
+  CANSparkMax Left2 = new CANSparkMax(Constants.DRIVETRAIN_LEFT2_ID, MotorType.kBrushless);
+  CANSparkMax Rightmaster = new CANSparkMax(Constants.DRIVETRAIN_RIGHTMASTER_ID, MotorType.kBrushless);
+  CANSparkMax Right1 = new CANSparkMax(Constants.DRIVETRAIN_RIGHT1_ID, MotorType.kBrushless);
+  CANSparkMax Right2 = new CANSparkMax(Constants.DRIVETRAIN_RIGHT2_ID, MotorType.kBrushless);
```

Comment:
These should have "private" on them.

Style question - Do we want new operations here at the variable define or below in the constructor.  So far, I have seen both but we should pick one for consistency.

Reply:
For reference, working SparkMAX/NEO code in 2020-Testing, branch **template-NCR** declares the variable names at the top of the file and initializes (new) in the constructor. It shouldn't matter  _where_ this happens but just in case you need inspiration from a working solution for structure.

Reply:
Another style comment: variable names should be heads down camel case.  Leftmaster should then be leftMaster and Left1 should be left1.  Classes are heads up camel case so in this case, Drivetrain works.  Drivetrain is a compound word so Drivetrain, not DriveTrain.